lib/internal/modules/run_main.js

```diff
@@ -162,7 +157,34 @@ function runEntryPointWithESMLoader(callback) {
 function executeUserEntryPoint(main = process.argv[1]) {
   const resolvedMain = resolveMainPath(main);
   const useESMLoader = shouldUseESMLoader(resolvedMain);
-  if (useESMLoader) {
+
```

Comment:
Can you also update the description of `executeUserEntryPoint` jsdoc and reflect the change in here?